installation/resources/installer.yaml

```diff
@@ -233,7 +233,7 @@ spec:
       serviceAccountName: kyma-installer
       containers:
       - name: kyma-installer-container
-        image: eu.gcr.io/kyma-project/develop/installer:0fdc80dd
+        image: eu.gcr.io/kyma-project/kyma-installer:1.0.0-rc1
```

Comment:
installer and kyma-installer are 2 different images, please talk to @Tomasz-Smelcerz-SAP

Reply:
We are aware of this, now there should be kyma-installer as this is also release artifact.